test/aspect-ratio.js

```diff
@@ -0,0 +1,10 @@
+import { setup } from './_helpers.js'
+import { expect, test } from 'vitest'
+
+setup()
+
+test('it generates backported aspect-ratios', async ({ uno }) => {
+  const classes = ['aspect-16/9', 'aspect-1/1', 'aspect-4/3', 'aspect-2/1']
+  const { css } = await uno.generate(classes)
+  expect(css).toMatchSnapshot()
```

Comment:
Would it make sense to add a test for the values that we do not support to test that we do not generate any styling at all then?